src/Week1/HW01Tests.hs

```diff
@@ -4,7 +4,7 @@
 
 module HW01Tests where
 
-import HW01
+import Week1.HW01
```

Comment:
The module names should reflect the directory path starting from what you have in `hs-source-dirs` field of the library section in your cabal file. With this name stack / cabal will be able to discover the file if you run `stack repl` (or most other stack commands for that matter) from the root of your project.